modin/pandas/test/test_groupby.py

```diff
@@ -1340,7 +1371,8 @@ def test(grp):
 
         return test
 
-    # issue-#3252
+    # issue-#3252, https://github.com/pandas-dev/pandas/issues/52760
+    """
```

Comment:
please do not disable code using block-comment, it isn't possible to see in diffs or when `git blame`-ing; comment each line if you don't want to remove the code